src/database/seeders/20201118073216-demo-user.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('Users', [
+      {
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'john@gmail.com',
+        nationalId: 1199480012023222,
+        password: '123abc',
+        phone: '0788767562',
+        language: 'en',
+        role: 'admin',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        firstName: 'Guevara',
+        lastName: 'Che',
+        email: 'che@gmail.com',
+        nationalId: 1200280012023224,
+        password: '123abc',
+        role: 'driver',
+        phone: '0788767561',
+        language: 'en',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        firstName: 'Egide',
+        lastName: 'Isabwe',
+        email: 'isabwe@gmail.com',
+        nationalId: 1199580012023225,
```

Comment:
This looks like a legit ID number, don't you think it is not a good idea?

Reply:
Also worked on that. thank you